nflows/utils/torchutils.py

```diff
@@ -160,8 +161,8 @@ def get_temperature(max_value, bound=1 - 1e-3):
     return temperature
 
 
-def notinfnotnan(x):
-    return torch.all(~torch.isnan(x)) and torch.all(~torch.isinf(x))
+def notinfnotnan(x: torch.Tensor) -> torch.Tensor:
```

Comment:
Nice: `isfinite` is a good way to do this. Happy for uses of `notinfnotnan` to be replaced with it at some point.